src/components/SearchBar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const SearchBar = ({ value, setValue }) => {
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
+
+  const clearText = (event) => {
+    event.preventDefault();
+    setValue('');
+  };
+
+  return (
+    <div>
+      <input
+        type="text"
+        placeholder="Search list.."
+        value={value}
+        onChange={handleChange}
+      />
+      {value ? (
+        <button type="reset" onClick={clearText} value="Reset">
+          X
+        </button>
+      ) : null}
```

Comment:
Here you can also skip the ternary and use `&&` since you don't return anything when you don't have a value :slightly_smiling_face:

Reply:
Ah great thanks Golfo! I've amended it now :)